crates/gitbutler-repo/src/rebase.rs

```diff
@@ -128,12 +128,12 @@ fn commit_unconflicted_cherry_result<'repo>(
         .context("failed to find commit")
 }
 
-fn commit_conflicted_cherry_result<'l>(
-    ctx: &'l CommandContext,
+fn commit_conflicted_cherry_result<'repository>(
```

Comment:
How do you feel about sticking to the rust conventions and idiomatic lifetime naming? It's technically not an abbreviation...so don't worry about calling it `'a`

Reply:
Git2 has the lifetime named `'repo`

<img width="1299" alt="Screenshot 2024-08-22 at 18 21 12" src="https://github.com/user-attachments/assets/69345694-9618-489a-9133-81abbd8323d9">

Reply:
The lifetime in this function is related to the repository, so it makes sense to name it accordingly

Reply:
Fair enough 🤷. I am mostly trying to imitate the style of code in the standard lib.